include/ppx/geometry.h

```diff
@@ -261,6 +296,10 @@ class Geometry
     void AppendTriangle(const TriMeshVertexData& vtx0, const TriMeshVertexData& vtx1, const TriMeshVertexData& vtx2);
     void AppendEdge(const WireMeshVertexData& vtx0, const WireMeshVertexData& vtx1);
 
+    // Directly overwrites index/vertex buffers
+    void SetIndexBuffer(const Geometry::Buffer& indexBuffer) { mIndexBuffer = indexBuffer; }
```

Comment:
There are implementation details on top of this class: "Implementation Notes: Do not modify the vertex buffers directly, use the Append* functions"
Is there a reason to do this?